MonamiYP/portfolio
# Feature/MYP-2 homepage

### What has been done in this PR
The goal of this PR is to begin the basic setup for the project using Angular. Tailwind CSS was also installed to make styling easier. Furthermore, a home page was created.

### Jira tickets/ links

- [MYP-2](https://monamiyp.atlassian.net/jira/software/projects/MYP/boards/3?selectedIssue=MYP-2)
- Link to [Figma](https://www.figma.com/file/a8Uhb6JuC8ddVflHCDueCJ/Portfolio?node-id=0%3A1) design

### How to test this PR

- Run the Angular project with _ng serve --open_
- Check that the home page displays three title texts
